src/main/kotlin/com/github/avrokotlin/avro4k/encoder/UnionEncoder.kt

```diff
@@ -16,7 +16,10 @@ import org.apache.avro.Schema
 class UnionEncoder(private val unionSchema : Schema,
                    override val serializersModule: SerializersModule,
                    private val callback: (Record) -> Unit) : AbstractEncoder() {
-
+   override fun encodeString(value: String){
+      //No need to encode the name of the concrete type. The name will never be encoded in the avro schema.
+      println(value)
```

Comment:
Did you mean to leave this `println` here @thake ?

Reply:
Thanks, just needed for a breakpoint. :) I will remove it